db/error_handler.cc

```diff
@@ -338,7 +338,7 @@ Status ErrorHandler::SetBGError(const IOStatus& bg_io_err,
   }
   if (BackgroundErrorReason::kManifestWrite == reason) {
     // Always returns ok
-    db_->DisableFileDeletionsWithLock();
+    db_->DisableFileDeletionsWithLock().PermitUncheckedError();
```

Comment:
Please add a comment that this always returns OK so the status here is unimportant (or change the method to return a void...)

Reply:
Sure